### ISSUE TITLE
Issue - Incorrect release version

### ISSUE THREAD

**Issue by kumarsiyer** [cloud-custodian/cloud-custodian#5277]
https://github.com/cloud-custodian/cloud-custodian/releases/tag/0.8.46.0

```
pip show c7n
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Name: c7n
Version: 0.8.46.0
Summary: Cloud Custodian - Policy Rules Engine
Home-page: https://github.com/cloud-custodian/cloud-custodian
Author: None
Author-email: None
License: Apache-2.0
Location: [REDACTED]
Requires: python-dateutil, jsonpatch, PyYAML, botocore, boto3, urllib3, argcomplete, jsonschema, certifi, tabulate
Required-by: c7n-org
```

However when you run  

> custodian version

 it still shows as **0.8.45.4**

**Comment by kapilt** [comment]
thanks for the report, put up a branch to address, hoping to automate releases with the frozen dependency stuff.

you should try getting off python 2 ;-)

**Comment by kapilt** [comment]
#5279 will also fix this more permanently.

**Comment by kumarsiyer** [comment]
Hahaha yeah... I know should stop using python2 :) and thanks for the quick fix.

**Comment by kapilt** [comment]
released 0.8.46.1 also folks in gitter reported another py2 issue which is also resolved.
